docs/ergo-cli.md

```diff
@@ -147,33 +349,27 @@ If what you are trying to write is too long to fit on one line, you can use `\`
 You can load CTO and Ergo files to use in your REPL session. Once the REPL is launched you will have to import the corresponding namespace. For instance, if you want to use the `compoundInterestMultiple` function defined in the `./examples/promissory-note/money.ergo` file, you can do it as follows:
 
 ```ergo
-$ ergotop ./examples/promissory-note/money.ergo 
-08:45:26 - info: Logging initialized. 2018-09-25T12:45:26.481Z
+$ ergotop ./examples/promissory-note/logic/money.ergo
+Welcome to ERGOTOP version 0.20.0
+
 ergo$ import org.accordproject.ergo.money.*
-ergo$ return compoundInterestMultiple(0.035, 100.0)
+ergo$ return compoundInterestMultiple(0.035, 100)
 Response. 1.00946960405 : Double
-ergo$ 
 ```
 
 ### Calling contracts
 
 To call a contract, you first needs to _instantiate_ it, which means setting its parameters and initializing its state. You can do this by using the `set contract` and `call init` commands respectively. Here is an example using the `volumediscount` template:
 
 ```ergo
-$ ergotop ./examples/volumediscount/model.cto ./examples/volumediscount/logic.ergo 
+$ ergotop ./examples/volumediscount/model/model.cto ./examples/volumediscount/logic/logic.ergo
 ergo$ import org.accordproject.cicero.contract.*
 ergo$ import org.accordproject.cicero.runtime.*
 ergo$ import org.accordproject.volumediscount.*
-ergo$ set contract VolumeDiscount over TemplateModel{ \
-  ...   firstVolume: 1.0, \
-  ...   secondVolume: 10.0, \
-  ...   firstRate: 3.0, \
-  ...   secondRate: 2.9, \
-  ...   thirdRate: 2.8 \
-  ... }
-ergo$ call init(Request{})
+ergo$ set contract VolumeDiscount over VolumeDiscountContract {firstVolume: 1.0, secondVolume: 10.0, firstRate: 3.0, secondRate: 2.9, thirdRate: 2.8, contractId: "0", parties: none }
+ergo$ call init()
```

Comment:
changed because `call init(Request{})` did not work.

All the commands below this section does not work.